eos_data_distribution/store/simple_store.py

```diff
@@ -42,8 +42,7 @@ def __init__(self, base, prefix='/',
         self.dirs = dict()
         self.dirpubs = defaultdict(lambda: {})
 
-        if base:
-            self.publish_all_names(base)
+        self.publish_all_names(self.base)
```

Comment:
it's because a former commit made base mandatory, so that if base doesn't make sense.
also, yes we need to have the same paths (realpath()) or we get confused.